lms/static/sass/views/_login-register.scss

```diff
@@ -111,51 +97,35 @@ $sm-btn-linkedin: #0077b5;
 
             .text {
```

Comment:
Oh, I meant leave the rule as-is, but nest it inside &.lines since it should only apply when there is a line.